Sources/SWBTaskExecution/TaskActions/ClangCompileTaskAction.swift

```diff
@@ -301,6 +301,19 @@ public final class ClangCompileTaskAction: TaskAction, BuildValueValidatingTaskA
                         outputDelegate.emitOutput("Failed frontend command:\n")
                         outputDelegate.emitOutput(ByteString(encodingAsUTF8: commandString) + "\n")
                     }
+
+                    if case .some(.exit(.uncaughtSignal, _)) = outputDelegate.result {
```

Comment:
We treat some signals as cancellation; should we avoid creating reproducers in that case?

Reply:
If we have a cancellation, does `TaskProcessDelegate.commandResult == .cancelled`? We can generate a reproducer only when `lastResult == .failed` to avoid useless reproducers. I think in my testing I've encountered signals to some compilations after the build encountered crashes for a different file.